wasm/bench/v8_bench_test.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"os"
+	"testing"
+
+	"rogchap.com/v8go"
+)
+
+func mustRead(tb testing.TB, path string) []byte {
+	tb.Helper()
+	data, err := os.ReadFile(path)
+	if err != nil {
+		tb.Fatalf("ReadFile %q: %v", path, err)
+	}
+	return data
+}
+
+func newV8Instance(b *testing.B) *v8go.Context {
+	iso := v8go.NewIsolate()
+	ctx := v8go.NewContext(iso)
+	return ctx
+}
+
+func BenchmarkV8_RunJS_Separate(b *testing.B) {
+	poly := mustRead(b, "testdata/polyfill.bundle.js")
+	bund := mustRead(b, "testdata/bundle.js")
+
+	ctx := newV8Instance(b)
+	defer ctx.Isolate().Dispose()
+
+	if _, err := ctx.RunScript(string(poly), "polyfill.js"); err != nil {
+		b.Fatalf("polyfill init error: %v", err)
+	}
+
+	b.ReportAllocs()
+	b.ResetTimer()
```

Comment:
This reset all timer and metrics losing all the work done prior it, don't think your number are actually meaningful.